### PR TITLE
Zyxel XGS4600 ssh/telnet

### PR DESCRIPTION
**Pull request by cppmonkey**
Whilst I've tried to include FTP, SSH and Telnet in the one file. Ultimately I've been unsuccessful, perhaps I'll find away in the near future.

The annoying thing is the strange escape character the switches return at the end of the prompt.

Unfortunately I only have access to XGS4600-32F switches for validation and currently the script assumes the switches have had stacking mode enabled.

The following command help identify stacking faults (chain vs ring)
_cmd 'show stacking'_

### CODE REVIEW

File: docs/Model-Notes/XGS4600-Zyxel.md [hunk 1]
```diff
@@ -20,6 +20,20 @@ input:
     passive: false
 ```
 
+
+## SSH/TelNet
+
+Below is the table from the XGS4600 CLI Reference Guide (Version 3.79~4.50 Edition 1, 07/2017)
+Take this table with a pink of salt, level 3 will not allow _show running-config_!
```
Comment (laf) [inline docs/Model-Notes/XGS4600-Zyxel.md]: `pink` -> `pinch`

**Comment by laf** [comment]
Thanks for this @cppmonkey. Can you please rebase this as the README has been updated since this PR.

File: docs/Model-Notes/README.md [hunk 2]
```diff
@@ -13,7 +13,6 @@ Arbor Networks|[ArbOS](ArbOS.md)|27 Feb 2018
 Arista|[EOS](EOS.md)|05 Feb 2018
 Huawei|[VRP](VRP-Huawei.md)|17 Nov 2017
 Juniper|[MX/QFX/EX/SRX/J Series](JunOS.md)|18 Jan 2018
-Netgear|[Netgear](Netgear.md)|11 Apr 2018
```
Comment (laf) [inline docs/Model-Notes/README.md]: Looks like the rebase has removed the Netgear model. Can you re-add that back in please.

File: docs/Model-Notes/XGS4600-Zyxel.md (hunk 1 above)
Comment (laf) [reply docs/Model-Notes/XGS4600-Zyxel.md]: This still needs updating :)

**Comment by cppmonkey** [comment]
Sorry, need a little more practice working with git

**Comment by cppmonkey** [comment]
@laf are you at #uknof40 by any chance?

**Comment by laf** [comment]
@cppmonkey I am indeed, are you?

**Comment by cppmonkey** [comment]
That I am, along with a few of my colleagues. I wondered if the hoody was merch or if you were a dev and then I saw your display picture. Uncanny, could it be!

**Comment by laf** [comment]
@cppmonkey I'm a maintainer so bought myself a hoodie. We've looked at a store before but aren't happy with the ones we tried. Can send you the link to order one though if you want.

**Comment by ytti** [comment]
I think we should address how to present tftp, telnet, ssh in same model. But seems beyond scope of this PR.

**Comment by cppmonkey** [comment]
Would love to have, but my Ruby skills are a little limited.
Been a few years since I programmed on a day to day basis.

@laf was hoping I'd catch up with you during pints 'n packets. Looks like you had a productive day, my inbox is rammed.

Awesome job you guys do! I'd become a maintainer, but not sure I'd have the time.

**Comment by laf** [comment]
> @laf was hoping I'd catch up with you during pints 'n packets. Looks like you had a productive day, my inbox is rammed.

Family time on Friday night so I bailed straight after the last talk.

**Pull request merged**